app/utils/report/test.py

```diff
@@ -47,6 +47,7 @@
     models.JOB_KEY,
     models.KERNEL_KEY,
     models.LAB_NAME_KEY,
+    models.MEASUREMENTS_KEY,
```

Comment:
This lists the fields that should be retrieved from the test groups collection.  The `models.MEASUREMENTS_KEY` is not an attribute of the test group model, so I don't think this is needed.  The measurements are part of the test case model, and the test case entries are added by the `_add_test_group_data` function.

Reply:
oky will remove it and try without